CONTRACT-TESTS.md

```diff
@@ -0,0 +1,20 @@
+# API Contract Tests
+
+It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
+in order to have an isolated environment.
+
+From the root directory of the project
+
+1. `virtualenv venv`
+2. `source venv/bin/activate` to turn on the virtualenv
+2. `./venv/bin/pip install -r test-requirements.txt`
```

Comment:
These docs seem to collide with our normal setup docs. For example, we suggest naming the virtualenv `venv` as well, and the two aren't compatible. It probably also needs to be called out that these need to be Python 2.